shepherd_bms/src/calcs.cpp

```diff
@@ -135,4 +132,55 @@ void calcContCCL(AccumulatorData_t *bmsdata)
     } else {
         bmsdata->chargeLimit = TEMP_TO_CCL[maxResIndex];
     }
+}
+
```

Comment:
this big guy needs some comments

Reply:
Thats on me. Should I add them as suggestions in this review?

Reply:
sorry for late reply but yeah that might be easiest, doesnt matter too much since they would be easy to add after